Mlem/Views/Tabs/Posts/Components/Shared/Links/User Profile Label.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  User Profile Label.swift
+//  Mlem
+//
+//  Created by Jake Shirley on 6/21/23.
+//
+
+import SwiftUI
+import CachedAsyncImage
+
+struct UserProfileLabel : View {
+    @AppStorage("shouldShowUserAvatars") var shouldShowUserAvatars: Bool = true
+    
+    @State var account: SavedAccount
+    @State var user: APIPerson
+    
+    // Extra context about where the link is being displayed
+    // to pick the correct flair
+    @State var postContext: APIPostView? = nil
+    @State var commentContext: APIComment? = nil
+    
+    static let developerNames = [
+        "lemmy.ml/u/lFenix",
```

Comment:
I guess we should remove this, as David removed himself from the about page before leaving 😞

Reply:
Done